src/main/java/edu/utdallas/davisbase/catalog/DavisBaseColumnsTableColumn.java

```diff
@@ -1,29 +1,35 @@
 package edu.utdallas.davisbase.catalog;
 
-import static java.lang.String.format;
-
 import edu.utdallas.davisbase.DataType;
 
+import static java.lang.String.format;
+
 /**
  * An enumerated column schema for the {@link CatalogTable#DAVISBASE_COLUMNS davisbase_columns}
  * catalog table.
  */
 public enum DavisBaseColumnsTableColumn implements CatalogTableColumn {
-  ROWID            (DataType.INT,     false),
-  TABLE_NAME       (DataType.TEXT,    false),
-  COLUMN_NAME      (DataType.TEXT,    false),
-  DATA_TYPE        (DataType.TEXT,    false),
-  ORDINAL_POSITION (DataType.TINYINT, false),
-  IS_NULLABLE      (DataType.TEXT,    false);
+  ROWID            (DataType.INT,     false, false, false),
+  TABLE_NAME       (DataType.TEXT,    false, true, false),
+  COLUMN_NAME      (DataType.TEXT,    false, false, false),
+  DATA_TYPE        (DataType.TEXT,    false, false, false),
+  ORDINAL_POSITION (DataType.TINYINT, false, false, false),
+  IS_NULLABLE      (DataType.TEXT,    false, false, false),
+  IS_UNIQUE      (DataType.TEXT,false, false, false),
+  IS_PRIMARYKEY      (DataType.TEXT,false, false, false);
```

Comment:
```suggestion
  IS_PRIMARYKEY    (DataType.TEXT, false, false, false);
```

Also, let's change `IS_PRIMARYKEY` to `IS_PRIMARY_KEY` for consistency. It will need to be updated wherever it's used as well, but that should be a single click of a button in IntelliJ (something like "Refactor - Rename").